modules/custom/az_news/az_news_export/src/Plugin/views/row/AZNewsDataFieldRow.php

```diff
@@ -77,7 +77,12 @@ public function render($row) {
                 $item[$key] = $image_style->buildUrl($uri);
               }
             }
-
+            if (!empty($media->field_media_az_image->alt)) {
+              $item['alt'] = $media->field_media_az_image->alt;
+            }
+            if (!empty($media->field_media_az_image->title)) {
+              $item['title'] = $media->field_media_az_image->title;
+            }
```

Comment:
This impacts the schema of the API because previously these elements consisted only of URL propeties. This may impact the Trellis integration that runs on this data. We should check and make sure this won't cause an issue.

This approach seems like a sound place for these elements to me, though.

Reply:
Good point.